### PR TITLE
post(ga-gtm): new post

### PR DESCRIPTION
**Pull request by bbbenja**

### CODE REVIEW

File: site/content/blog/2018-03-12-GA-GTM.adoc [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+cover: ga-gtm-banner
+date: '2018-04-09'
+tags:
+- GA
+- GTM
+title: 'Google Analytics via Google Tag Manager'
+url: /2018/04/09/ga-gtm/
+---
+
+C'est en branchant https://www.facebook.com/business/a/facebook-pixel[PIXEL] sur un projet que j'ai enfin compris et vraiment utilisé *Google Tag Manager*.
```
Comment (CedricGatay) [inline site/content/blog/2018-03-12-GA-GTM.adoc]: I think you could put the full name the first time "Facebook PIXEL" (I clicked to understand what it was)

File: site/content/blog/2018-03-12-GA-GTM.adoc [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+:lightboxGroup: 0
+:lightboxImage: 2018-03-22_GA-GTM/debug_panel
+:lightboxImageTitle: Panel de debug
+
+include::site/static/lightbox.adoc[]
+
+*Google Tag Manager* se découpe en trois grandes parties :
+
+* les balises (tag) : une balise représente du code Javascript que l'on va pousser et exécuter dans notre conteneur.
```
Comment (CedricGatay) [inline site/content/blog/2018-03-12-GA-GTM.adoc]: put english words in _italic_

File: site/content/blog/2018-03-12-GA-GTM.adoc [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+* les déclencheurs (triggers) : les déclencheurs sont des conditions permettant de pousser ou non nos balises.
+* les variables sont ce qu'on pourra interpréter dans notre balise. Il y en a certaines déjà définies et d'autres que l'on peut construire.
+
+Pour ajouter *Google Analytics* à notre application, il va falloir utiliser ces trois composants.
+
+
+== 2ème étape : Ajouter Google Analytics
+
+Normalement pour ajouter *Google Analytics* à votre application, vous auriez dû (de la même manière qu'on a ajouté *GTM* juste avant), suivre les instructions de *Google Analytics* à la création du projet sur celui-ci et copier-coller un bout de code pour l'activer, mais vous l'avez compris, ce n'est pas ce qu'il va se passer vu qu'on ne va plus toucher au code.
```
Comment (CedricGatay) [inline site/content/blog/2018-03-12-GA-GTM.adoc]: This is a very long sentence, consider breaking it to ease understanding.

File: site/content/blog/2018-03-12-GA-GTM.adoc [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+Premièrement, vous devez quand même créer le projet sur https://analytics.google.com/ et garder dans un coin le `trackingId`, ensuite, tout se passe dans *Google Tag Manager*.
+
+Le principe premier de *Google Analytics* est de traquer les différentes pages consultées par l'utilisateur.
+Cette information se trouve normalement dans la variable `Page Path` qui fait partie des variables activées par défaut à la création du conteneur.
+Mais si comme nous vous faites beaucoup d'*Angular*, vous avez à faire avec des urls du type `http://localhost:9000/#/home`.
```
Comment (CedricGatay) [inline site/content/blog/2018-03-12-GA-GTM.adoc]: => "Mais si, comme nous, "

File: site/content/blog/2018-03-12-GA-GTM.adoc [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+ Pour ce faire, vous devez activer le mode debug en cliquant sur `PRÉVISUALISER` si ce n'est pas déjà fait et lancer votre application.
+ Dans le panel de debug, vous verrez normalemement la liste des différents `déclencheurs` sur la gauche, et lorsque vous cliquez dessus, vous pouvez voir quelles balises ont été activées et dans l'onglet `Variables`, la valeur des différentes variables (dont votre `New History Fragment`).
+
+:lightboxGroup: 0
+:lightboxImage: 2018-03-22_GA-GTM/debug_panel_1
+:lightboxImageTitle: Panel de debug
+
+include::site/static/lightbox.adoc[]
+
+Il ne reste plus qu'à publier vos modifications pour qu'elles soient actives sur tous vos environnements en cliquant sur `ENVOYER` et `PUBLIER`.
```
Comment (CedricGatay) [inline site/content/blog/2018-03-12-GA-GTM.adoc]: There is a lot of mix between french and english wording in this block, we should maybe put everything in english ?

File: site/content/blog/2018-03-12-GA-GTM.adoc [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+
+== Avantages de GTM
+
+GTM propose de multiples fonctionnalités à travers les balises, les déclencheurs et les variables.
+Vous pouvez par exemple déclarer un tableau de variables qui seront différentes suivant les environnements.
+Vous pouvez récupérer des valeurs de votre site grâce à des règles CSS afin de les pousser par exemple vers GA et tout ça sans avoir à relivrer votre application.
+
```
Comment (CedricGatay) [inline site/content/blog/2018-03-12-GA-GTM.adoc]: This block is small if GTM is so useful, maybe add another example could prove its usefullness

File: site/content/blog/2018-03-12-GA-GTM.adoc (hunk 5 above)
Comment (bbbenja) [reply site/content/blog/2018-03-12-GA-GTM.adoc]: This is what we have on google tag manager dashboard (if the browser is in french). Variables are always in english.

Comment (CedricGatay) [reply site/content/blog/2018-03-12-GA-GTM.adoc]: Looks ok then :)

**Pull request merged**